libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,9 +204,28 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getIetfLocale = (ietfLocale) => {
-  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
-  return nonStandardLocaleMap[ietfLocale] || ietfLocale;
+const getUniversalNavLocale = () => {
+  const { locale: { prefix: configPrefix = '', ietf = '' } = {} } = getConfig() || {};
+  const prefix = configPrefix.replace('/', '');
```

Comment:
Are there consumers using slashes in their prefixes?

Reply:
no, but that's how the prefix is returned. similar code is already on this branch https://github.com/adobecom/milo/blob/gnav/libs/blocks/global-navigation/global-navigation.js#L446